app/models/plog.go

```diff
@@ -14,3 +14,12 @@ type Plog struct {
 func (plog Plog) Description() string {
 	return plog.Text
 }
+
+// Used for uploading
+type PlogData struct {
+	Text         string
+	Protagonista int
+	Autor        int
+	Titol        string
+	Data         int64 // Unix epoch
```

Comment:
`Data` is misleading. Can we use `Time` or `Timestamp` instead?

Reply:
Well these fields are in Catalan so it would look a bit off.